YMOverrideTestSupport.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
     s.name             = 'YMOverrideTestSupport'
-    s.version          = '2.3.0'
+    s.version          = '2.4.0'
```

Comment:
Don't need to increment because we didn't actually release 2.3 (or 2.2 for that matter)